virttest/utils_disk.py

```diff
@@ -372,15 +373,22 @@ def create_partition_linux(session, did, size, start,
     :param start: partition beginning at start. e.g. 0M
     :param part_type: partition type, primary extended logical
     :param timeout: Timeout for cmd execution in seconds.
+    :return: the created partition label.
     """
     size = utils_numeric.normalize_data_size(size, order_magnitude="M") + "M"
     start = utils_numeric.normalize_data_size(start, order_magnitude="M") + "M"
     end = str(float(start[:-1]) + float(size[:-1])) + size[-1]
     partprobe_cmd = "partprobe /dev/%s" % did
-    mkpart_cmd = 'parted -s "%s" mkpart %s %s %s'
-    mkpart_cmd %= ("/dev/%s" % did, part_type, start, end)
-    session.cmd(mkpart_cmd)
+    mkpart_cmd = 'parted -s "/dev/%s" mkpart %s %s %s print' % (did, part_type,
+                                                                start, end)
+    output = session.cmd(mkpart_cmd)
+    partition = did + re.findall(r'\s+(\d+)\s+\d+', output, re.M)[-1]
```

Comment:
It's not suitable for nvme disk:
e.g.
nvme0n1                          259:0    0 745.2G  0 disk
├─nvme0n1p1                      259:1    0     1G  0 part
└─nvme0n1p2                      259:2    0 744.2G  0 part
  ├─rhel_bootp--73--225--39-swap 253:2    0     4G  0 lvm
  ├─rhel_bootp--73--225--39-home 253:3    0 690.2G  0 lvm
  └─rhel_bootp--73--225--39-root 253:4    0    50G  0 lvm